### PR TITLE
feat(SimpleFileUpload): consumed Penta updates

### LINKED ISSUE #9996: Consume core Penta updates: Single file upload
Follow up to core issue https://github.com/patternfly/patternfly/issues/6209

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #9996
codemod: https://github.com/patternfly/pf-codemods/issues/551

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/components/FileUpload/FileUploadField.tsx [hunk 1]
```diff
@@ -138,16 +143,15 @@ export const FileUploadField: React.FunctionComponent<FileUploadFieldProps> = ({
               name={`${id}-filename`}
               aria-label={filenameAriaLabel}
               placeholder={filenamePlaceholder}
-              aria-describedby={`${id}-browse-button`}
               value={filename}
             />
           </InputGroupItem>
           <InputGroupItem>
             <Button
-              id={`${id}-browse-button`}
```
Comment (thatblindgeye) [inline packages/react-core/src/components/FileUpload/FileUploadField.tsx:147]: We could keep this in, but if we're going to expose an ID on the browse button then we should consider allowing an ID to be passed to all other input elements within the component. For now I included this in the codemod issue https://github.com/patternfly/pf-codemods/issues/551

**Review by nicolethoen — COMMENTED** [review]
This will need a codemod, right? To deal with the helper text change?

**Comment by thatblindgeye** [comment]
@nicolethoen good point. I updated codemod https://github.com/patternfly/pf-codemods/issues/551 to mention providing a warning that consumers should use the new FileUploadHelperText sub-component (right now we only have examples that show using a FormHelperText as a separate piece).

**Review by wise-king-sullyman — APPROVED** [review]
🔥

**Review by nicolethoen — APPROVED** [review]

**Review by lboehling — APPROVED** [review]
very niiice

**Review by srambach — APPROVED** [review]
LGTM 🗄️

**Pull request merged**